CONTRIBUTING.md

```diff
@@ -1,57 +1,189 @@
+- Anyone may informally review a pull request and make comments or
+  suggestions.
+- Only 18F staff may merge pull requests (please do not merge your own pull
+  requests, ask for someone else to review and do so).
+
+
+## 18F team information and practices
+There are currently two teams at 18F working in the FEC repositories: the data
+team and the legal team. Both teams have agreed to sync their sprint schedules
+and align their development efforts to coincide with combined bi-weekly
+releases that ship to production on the Wednesday after a two-week sprint. We
+will also share a calendar in order to keep track of important sprint dates
+and events.
+
+In addition, we've committed to setting aside some time during each sprint
+when we all work on code reviews of open pull requests together. This will
+help bring new developers up-to-speed on the various code bases and ensure
+folks are comfortable reviewing code across the teams. As everyone's comfort
+increases we can consider making this activity less structured.
+
+
+### Demos
+We have agreed to continue using our development site for demos with folks at
+the FEC. Everyone should feel empowered to demo their work as needed,
+especially as this provides valuable and timely feedback. We have also created
+a new space for us for testing and demoing specific features that is not
+auto-deployed to.
+
+While we routinely demo our work on the development site, the FEC folks will
+conduct full reviews of the work slated for a release on our staging site.
+
+
+### Deployment- and release-related information
+The betaFEC project requires a steady release schedule with contents of
+releases planned ahead of time due to project requirements. Since both teams
+have agreed to synchronize their sprint schedules, this will make it easier
+for us to plan for each release after a sprint is completed.
+
+In general, we are committed to creating and deploying a release branch on the
+Monday after a sprint is completed. This will require us to have everything
+intended to be ready for the release merged to the `develop` branch by the
+second Thursday in a sprint. We are allowing ourselves an extra day at the end
+to provide us with the bandwidth needed to address any issues, merge
+conflicts, etc.
+
+This release branch will be deployed to our `stage` environment for FEC folks
+to review until we are ready to do a production deployment, which we have
+agreed will be that Wednesday pending any issues found. When the branch is
+created we have agreed to let folks know in the appropriate Slack channels
+ahead of time to prevent surprises.
+
+We currently have automatic deployments take place whenever code is
+merged/pushed to the `develop` (to our `dev` space) or `release/*` (to our
+`stage` space) branches. We have created a separate `feature` space in Cloud
+Foundry that is not auto-deployed to so that we may work on and test specific
+features without clobbering each other.
+
```

Comment:
We are also setting up s "feature" environment where long lead time before committing to dev type features can be deployed to and tested without being disrupted by the frequent dev merges.  May make sense to plug that here?